### DISCUSSION TITLE
problems with hamoid VideoExport lib

### DISCUSSION THREAD

**Discussion by Maltiec** [py5coding/py5generator#288] [Q&A] [answered]
Today I've got several problems with VideoExport lib by Hamoid (it is still works fine on Ubuntu, but Windows 10 and Mac OS seems to be broken). What other libraries do we have for exporting video?

**Comment by hx2A** [comment, marked as answer]
What are you trying to do? Make a recording of a running Sketch? When I do this I will save frames using `py5_tools.save_frames()` and then encode the image sequence with ffmpeg.

https://py5coding.org/reference/py5tools_save_frames.html
